mass/core/models.py

```diff
@@ -42,3 +43,28 @@ class Filter(BaseModel):
 
     key: str = Field(..., description="The field to filter")
     value: str = Field(..., description="The value the field must match")
+
+
+class FacetOptions(BaseModel):
+    """Represents the facet options"""
+
+    option: Union[str, None] = Field(..., description="The value of the facet option")
+    count: int = Field(default=1, description="The number of occurrences of the option")
+
+
+class Facet(BaseModel):
+    """Represents a facet"""
+
+    key: str = Field(..., description="The raw facet key, such as study.type")
+    name: str = Field(default="", description="The user-friendly name for the facet")
+    options: list[FacetOptions] = Field(
+        ..., description="The list of options for the facet"
+    )
```

Comment:
Could alternatively also be modeled as a dictionary where the keys correspond to the options and the values correspond to the counts.

Reply:
@KerstenBreuer I agree, and that might be nicer. I will look at how to change the query to reflect this.

Reply:
Fixed in [74d86c6](https://github.com/ghga-de/mass/pull/5/commits/74d86c6ebc235d3faee0fded66bc3434435bace3)